src/components/UserListSkeletonView.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Since this is extremely similar to `OptionsListSkeletonView`, I'm wondering if it would be better to just pass a prop like `isTableList` to `OptionsListSkeletonView`, which would keep things more DRY. Any thoughts?

Reply:
@jjcoffee  Sorry, you mean that we should remove the UserListSkeletonView component, then handle it in OptionListSkeletonView by passing prop ```isUserList```, right?

Reply:
Yeah I think that's tidier, unless you have any other thoughts! I think `isTableList` as a prop name fits better here as we're only styling the list as a table, rather than changing the layout from the base layout to fit something specific for a user list.

Reply:
@jjcoffee In the main branch, we already have ```TableListItemSkeleton```, ```ItemListSkeletonView```, and ```OptionListSkeletonView``` (which is based on ```ItemListSkeletonView```). In this PR, we just added a new skeleton component, ```UserListSkeletonView```. To maintain consistency with other skeleton components, we should keep ```UserListSkeletonView``` as is. If we plan to let ```OptionListSkeletonView``` handle ```UserListSkeletonView``` by passing ```isUserList```, we should apply the same approach to the other skeleton components like ```TableListItemSkeleton``` and ```ItemListSkeletonView```.

Reply:
My point is that `UserListSkeletonView` is almost identical to `OptionListSkeletonView`, so I'm not sure how useful it is to split it out into a new component since it doesn't apply any layout changes, just a few style tweaks (as far as I can tell). `OptionListSkeletonView` uses `TableListItemSkeleton` which uses `ItemListSkeletonView`, so I'm not sure your point applies here. Unless I'm missing something!

Reply:
> OptionListSkeletonView uses TableListItemSkeleton which uses ItemListSkeletonView

Sorry, why you say that OptionListSkeletonView uses TableListItemSkeleton? Is there any mistake here?

Reply:
Haha oops sorry, I meant that they both use `ItemListSkeletonView`. We are displaying an `OptionList` here, so I don't know if it makes sense to add a new component as not all user lists should be displayed as tables. I think we are fundamentally saying "here's an option list, but we want it to look like a table". `TableListItemSkeleton` uses a completely different layout (not just styling), that's why it needs to be split into a separate component.